lib/version.js

```diff
@@ -88,8 +89,82 @@ function setCommand(version, pkg) {
 function set(v, project) {
   v = semver(v.toString());
   project.version(v);
+  setBlipIfOpted(project);
+}
+
+function setBlipIfOpted(project) {
+  var opted = false;
+  var blipScript = false;
+  var scriptPhases = {
+    'preinstall': project.scripts('preinstall'),
+    'install': project.scripts('install'),
+    'postinstall': project.scripts('postinstall')
+  };
+  var availPhases = [];
+  //check if the package opted for blip
+  hasOpted(project, scriptPhases, availPhases, opted, blipScript, function(hasOpted, hasBlipScript) {
+    opted = hasOpted;
+    blipScript = hasBlipScript;
+  });
+
+  if (opted && !blipScript) { //when package opted for Blip but doesn't have blip in scripts
+    if (availPhases.length) {
+      project.scripts(availPhases[0], 'node scripts/sl-blip.js'); //use first available script phase
+    } else { //if scripts exist, append sl-blip to existing postinstall script
+      var value = scriptPhases['postinstall'].toString() + ' && node scripts/sl-blip.js';
```

Comment:
This is probably how it would be done, but since nothing will likely trigger this code we probably don't need to worry about adding this complexity for handling it and should probably just abort instead.